.github/bin/run-clang-tidy.sh

```diff
@@ -32,17 +32,10 @@ hash ${CLANG_TIDY} || exit 2  # make sure it is installed.
 
 echo ::group::Build compilation database
 
-# First, build the compilation database.
-time bazel build :compdb > /dev/null 2>&1
+time $(dirname $0)/make-compilation-db.sh
 
 readonly EXEC_ROOT=$(bazel info execution_root)
```

Comment:
This is no longer needed here

Reply:
It is below when we edit the output to make more readable.